plugins/PostProcessingPlugin/scripts/FilamentChange.py

```diff
@@ -19,6 +22,14 @@ def getSettingDataString(self):
             "version": 2,
             "settings":
             {
+                "marlin": 
+                {
+                    "label": "Use Marlin Config?",
+                    "description": "Use configuration from Marlin Advanced Pause Feature!",
+                    "type": "bool",
+                    "default_value": true
```

Comment:
We don't want to default to `true` here. Existing FilamentChange users would see their Filament Change settings suddenly not working any more. It's also good to give this more time in the field before it gets to be the default. Not everyone has the feature in their printer either.